### PR TITLE
Fixes Deep-Fried Grabs and Display Case Grabs

### LINKED ISSUE #7212: You can put grabs into display cases
**Problem Description**:
What is the problem?
as the title says, you can put various grab levels into display cases
**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
-Grab someone
-Click on the open display case
-Be proud of your strength becoming more than just blunt force
https://i.gyazo.com/dca1220dd1934ec78552ab93843580e7.mp4

### LINKED ISSUE #7239: Deep Fried Grab
image

While fooling around in my test server, I came to found that when you go to dunk someone's head into the deep fryer, it will snag your grab intent and deep fry it...

Similar to the issue where grab intents could be placed into display cases. Other kitchen combat moves may have the same effect of taking in the grab as an "ingredient", though I've only observed it with the deep fryer so far. Able to be done with both the reinforced grab and kill grab, likely able to be done with the first level grab as well.

### PR DESCRIPTION
**Pull request by Twinmold93**
Fixes two issues: the ability to cook/deep-fry grabs and the ability to put grabs into display cases.

Fixes #7212
Fixes #7239

:cl: Twinmold
Fix: No longer able to put grabs into kitchen equipment (deep-fried grab).
Fix: No longer able to put grabs inside of display cases.
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
Should check whether the object has the `ABDSTRACT` flag.

You can still deep fry an Ei'Nath hand or something similar, I'd bet.

**Comment by scrubmcnoob** [comment]
jeez, you guys have never heard of modern art
my four years of schooling and 200,000 worth of debt has taught me that this is the pinnacle of modern art at its finest.

File: code/game/objects/structures/displaycase.dm [hunk 1]
```diff
@@ -256,6 +256,9 @@ var/global/list/captain_display_cases = list()
 			new /obj/machinery/constructable_frame/machine_frame(T)
 		qdel(src)
 		return
+	if(istype(W,/obj/item/weapon/grab))
```
Comment (Crazylemon64) [inline code/game/objects/structures/displaycase.dm]: Make this an `ABSTRACT` flag check, please!

Comment (Fox-McCloud) [inline code/game/objects/structures/displaycase.dm]: he could just have an `if(!user.drop_item()) return` check, instead.

Comment (Crazylemon64) [inline code/game/objects/structures/displaycase.dm]: Mm, that would do, yes

Comment (Twinmold93) [inline code/game/objects/structures/displaycase.dm]: `if(!user.drop_item())` does not work, in testing. How does the ABSTRACT check have to be set up?

**Comment by Twinmold93** [comment]
Fixed putting abstract items into the case. Also, you already can't put them in the kitchen equipment since they aren't listed as something that can be interacted with and there is no else notice.

**Pull request merged**